anthos-bm-gcp-terraform/versions.tf

```
@@ -15,7 +15,7 @@
  */
 
 terraform {
-  required_version = ">= v0.15.5, < 1.4" # this line should not change during a release
+  required_version = "< 1.5" # this line should not change during a release
```

It would be safer to preserve the `>=` constraint (`>= v0.15.5`).
```suggestion
  required_version = ">= v0.15.5, < 1.5" # this line should not change during a release
```